### DISCUSSION TITLE
MasterTemp400 RS485

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#540] [RS485] [answered]
Is there any advantage to connecting the MasterTemp400 via RS485 rather than breaking the "fireman's" connection?

**Comment by rstrouse** [comment, marked as answer]
Not really.  There are also some warts to using RS485.  First, the thermistor on the MT control board often does not match the thermistor for water temp.  Why they don't allow an external temp sensor for the thermostat in the control board is beyond me.  In the end, the guts of it are the same with the exception that it is in a thermowell that will eventually corrode.  Over the lifetime of the heater you will likely replace it.  

Second, the cooldown delays cannot be set by RS485 so configuration for the cooldown must be done at the heater.  Bear in mind MasterTemp claims that there is no need to run water through the heater during cooldown because the heat exchanger has a blower.  With that in mind I guess it makes no difference there.  Heater cooldowns at the controller level, maintain water flow through the heater for a set period after a call for heat.  Heaters without a blower must have water run through them or there is a danger that the heat buildup can water to boil inside the heater.

Finally, there are a ton of reports that the heater bypass valve does not work correctly on the control board.  It seems like they tied it to the blower so it will not rotate at the proper times.  Probably because the heater is never in charge of body modes.  This is an issue in the firmware so the fix will likely be to flash the firmware or buy a new control board.

I fully understand why the RS485 support is important for hybrid heaters and heatpumps because of the heat exchanger status, cooling modes, and dual operation.  But this is not the case for a gas heater and the processing at the control board has no knowledge as to what else is going on with the pool.

In the end you won't notice any appreciable difference that is positive.
